index.bs

```diff
@@ -90,47 +90,102 @@ we've prepared [a list of these questions in Markdown](https://raw.githubusercon
   and for what purposes is that exposure necessary?
 </h3>
 
-Just because information can be exposed to the web doesn’t mean that it
-should be. How does exposing this information to an origin benefit a user?
-Is the benefit outweighed by the potential risks?  If so, how?
+Just because information **can** be exposed to the web
+doesn’t mean that it **should** be.
+Does your feature expose information to origins?
```

Comment:
I think this needs clarification.

Reply:
I've revised this text, @torgo. What do you think?

Reply:
Rocks.